### ISSUE TITLE
Cannot import pythonjsonlogger.jsonlogger

### ISSUE THREAD

**Issue by gowdy** [nhairs/python-json-logger#29]
Was 3.2.0 meant to remove this?

from pythonjsonlogger.jsonlogger import JsonFormatter

Looks to still be deprecated from the release docs, nothing about it being removed (it says moved at the same time as deprecated).

**Comment by iTaybb** [comment]
It juts broke for my as well (using kopf).
I just rollbacked to 2.0 and it's working for me

**Comment by leroyshirto** [comment]
> It juts broke for me as well (using kopf). I just rollbacked to 2.0 and it's working for me

This has broken kopf for us too

**Comment by soxofaan** [comment]
`pythonjsonlogger.jsonlogger` seems to be removed (already in 3.1.0) with 49f69fbf65c85d01b858f3c7931de04d8851c815 for #9 

if I see correctly, an attempt was made to stay backward compatible:
https://github.com/nhairs/python-json-logger/blob/f96de2222ef60d4ab34b29d067db8b59642bf657/src/pythonjsonlogger/__init__.py#L22-L29

but that only seems to work with 
```python
from pythonjsonlogger import jsonlogger
```

not with 
```python
import pythonjsonlogger.jsonlogger
```

**Comment by nhairs** [comment]
Thanks for digging into that @soxofaan.

Short term workaround would be to pin your python-json-logger version.

Medium term would be to migrate the underlying code.

I will however see if I can fix this bug as I would like to keep backwards compatibility.

**Comment by nhairs** [comment]
I've uploaded a development version that should fix this.

Would someone be able to test using `python-json-logger==3.2.1.dev1` and get back to me on #33 if it resolved this issue?

**Comment by Rhiyo** [comment]
This broke our code as well.

quick fix for us was just to add

`python-json-logger<3.1.0` to our pinned dependencies.

**Comment by nhairs** [comment]
Hi all, I've released a fix for this issue (`python-json-logger==3.2.1`).

I don't currently intend to backport the changes, but could be convinced if someone has a compelling need.

**Comment by gowdy** [comment]
Thanks, that looks to have solved the issue.
